packages/formatter-codeframe/src/formatter.ts

```diff
@@ -49,8 +49,8 @@ const safeTrim = (txt: string, charsToRemove: number): boolean => {
 const codeFrame = (code: string, location: ProblemLocation) => {
     const codeInLines: Array<string> = `\n${code}`.split('\n');
     const whiteSpacesToRemove: number = countLeftWhiteSpaces(codeInLines[codeInLines.length - 1]);
-    const line: number = location.elementLine;
-    const column: number = location.elementColumn;
+    const line: number = location.elementLine || -1;
+    const column: number = location.elementColumn || -1;
```

Comment:
If I recall correctly, the values on `ProblemLocation` are zero-based so this needs an explicit type check to avoid converting `0` to `-1`:

```typescript
const line: number = typeof location.elementLine === 'number' ? location.elementLine : -1; 
const column: number = typeof location.elementColumn === 'number' ? location.elementColumn : -1;
```

Reply:
This should be fixed now